tests/snapshots/compile_file_o_all__self_ref.hvm.snap

```diff
@@ -3,7 +3,5 @@ source: tests/golden_tests.rs
 input_file: tests/golden_tests/compile_file_o_all/self_ref.hvm
 ---
 Errors:
-In definition 'Foo':
-  Definition is a reference to itself.
-In definition 'main':
-  Definition is a reference to itself.
+Inline:
```

Comment:
"While inlining the compiled program:" or "During inlining:".
Ideally we check this before so that we get actual function names, but it probably won't catch all cases that can happen here.